x-pack/plugins/secops/public/store/local/timeline/model.ts

```diff
@@ -17,15 +16,22 @@ export interface TimelineModel {
   range: Range;
   show: boolean;
   sort: Sort;
+  activePage: number;
+  itemsPerPage: number;
 }
 
 export const timelineDefaults: Readonly<
-  Pick<TimelineModel, 'dataProviders' | 'data' | 'range' | 'show' | 'sort'>
+  Pick<
+    TimelineModel,
+    'dataProviders' | 'data' | 'range' | 'show' | 'sort' | 'activePage' | 'itemsPerPage'
+  >
 > = {
   dataProviders: [],
-  data: mockECSData,
+  data: [],
```

Comment:
Note: I removed `mockECSData` for defaults and instead am using just an empty object. I think the `mockECSData` was temporary so it should be removed at this point as it is incorrect to be there instead of a default of empty data set